controllers/machineinventory_controller.go

```diff
@@ -234,7 +234,17 @@ func (r *MachineInventoryReconciler) updatePlanSecretWithReset(ctx context.Conte
 
 	logger.Info("Updating Secret with Reset plan")
 
-	checksum, resetPlan, err := r.newResetPlan(ctx)
+	var checksum string
+	var resetPlan []byte
+	var err error
+
+	unmanaged, unmanagedFound := mInventory.Annotations[elementalv1.MachineInventoryOSUnmanagedAnnotation]
+	if unmanagedFound && unmanaged == "true" {
```

Comment:
Missing some tests in `machineinventory_controller_test.go` to verify this branch.

Reply:
Thanks. Am in the process of re-writing these now. Will push an update later on. Appreciate the support!